certified-connectors/Signi/apiProperties.json

```diff
@@ -35,7 +35,7 @@
           "description": "Client (or Application) ID of the Azure Active Directory application.",
           "constraints": {
             "required": "false",
-            "hidden": true
+            "hidden": "true"

```

Comment:
Please check apiproperties validation errors and change the value of hidden parameter duly.